tests/Block/DeprecatedAuditBlockServiceTest.php

```diff
@@ -1,93 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-/*
- * This file is part of the Sonata Project package.
- *
- * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Sonata\DoctrineORMAdminBundle\Tests\Block;
-
-use Prophecy\Argument;
-use SimpleThings\EntityAudit\AuditReader as SimpleThingsAuditReader;
-use SimpleThings\EntityAudit\Revision;
-use Sonata\BlockBundle\Block\BlockContext;
-use Sonata\BlockBundle\Model\Block;
-use Sonata\BlockBundle\Test\BlockServiceTestCase;
-use Sonata\DoctrineORMAdminBundle\Block\AuditBlockService;
-
-/**
- * NEXT_MAJOR: Remove this class.
- *
- * @group legacy
- *
- * @author Marko Kunic <kunicmarko20@gmail.com>
- */
-class DeprecatedAuditBlockServiceTest extends BlockServiceTestCase
-{
-    private $simpleThingsAuditReader;
-    private $blockService;
-
-    protected function setUp(): void
-    {
-        parent::setUp();
-        $this->simpleThingsAuditReader = $this->prophesize(SimpleThingsAuditReader::class);
-
-        $this->blockService = new AuditBlockService(
-            'block.service',
-            $this->templating,
```

Comment:
IMO this test should not be removed, but this $this->templating should be fixed. It can be declared on this same class.

Reply:
And probably this will need to only execute this tests for block bundle 3, not for version 4.

Reply:
We can remove it becouse this arguments is not use directly in `AuditBlockService`. It is hide behind `renderResponse`. Result for templating and twig should be the same. This should be test by BlockBundle, not each Block which extends it.